assignment2.sql

```diff
@@ -363,3 +363,60 @@ WHERE NOT EXISTS
 
 DELETE FROM product p
 WHERE NOT EXISTS (SELECT * FROM services2products sp WHERE sp.product_id = p.id);
+
+
+-- HW4
+
+DELIMITER $$
+CREATE PROCEDURE product_details_by_name(IN product_name VARCHAR(255))
```

Comment:
I guess this file was mistakenly updated with mixed data from two other files, please revert its changes